boilerplate/app/models/helpers/useStores.ts

```diff
@@ -63,6 +63,7 @@ export const useInitialRootStore = (callback?: () => void | Promise<void>) => {
 
       // reactotron integration with the MST root store (DEV only)
       if (__DEV__) {
+        // @ts-ignore
         console.tron.trackMstNode(rootStore)
```

Comment:
not sure why this wasn't needed before, but I was getting typescript errors without.